.release-please-manifest.json

```diff
@@ -8,5 +8,6 @@
   "providers/unleash": "0.0.4-alpha",
   "providers/flipt": "0.0.2",
   "providers/configcat": "0.0.3",
-  "providers/statsig": "0.0.4"
-}
\ No newline at end of file
+  "providers/statsig": "0.0.4",
+  "tools/junit-openfeature": "0.0.0"
```

Comment:
I've never tried starting on `0.0.0` so I'm interested to see if this works (I guess it will).